src/control/cmd/daos_server/storage.go

```diff
@@ -120,22 +99,46 @@ func (cmd *storagePrepScmCmd) Execute(args []string) (err error) {
 		return errors.WithMessage(err, "initialising ControlService")
 	}
 
-	needsReboot, devices, err := svc.PrepareScm(server.PrepareScmRequest{
-		Reset: cmd.ResetScm,
-	})
-	if err != nil {
-		return err
+	cmd.log.Info("Preparing locally-attached storage...")
+
+	scanErrors := make([]error, 0, 2)
+
```

Comment:
I think there needs to be a sanity-check somewhere for having both NvmeOnly and ScmOnly set at the same time. In effect it would be the same as not supplying both of them, but we shouldn't allow it because it's ambiguous and could result from a scripting error.

Reply:
sure